src/test/java/seedu/address/model/person/InterviewTimeTest.java

```diff
@@ -23,12 +23,14 @@ public void isValidInterviewTime() {
         assertFalse(InterviewTime.isValidTime(" ")); // spaces only
         assertFalse(InterviewTime.isValidTime("^")); // only numbers and '/'
         assertFalse(InterviewTime.isValidTime("10/30/2003 3pm")); // invalid format
+        assertFalse(InterviewTime.isValidTime(("00/00/0000 1200"))); // invalid format
```

Comment:
Extra brackets here, might want to remove

Reply:
Thank you for spotting!